int32 count
int32[] data
